nntrainer/layers/rnncell.cpp

```diff
@@ -65,32 +63,44 @@ void RNNCellLayer::finalize(InitLayerContext &context) {
   const bool disable_bias =
     std::get<props::DisableBias>(*layer_impl_props).get();
 
+  NNTR_THROW_IF(std::get<props::Unit>(rnncell_props).empty(),
+                std::invalid_argument)
+    << "unit property missing for rnncell layer";
   const unsigned int unit = std::get<props::Unit>(rnncell_props).get();
+  const bool integrate_bias =
+    std::get<props::IntegrateBias>(rnncell_props).get();
   const nntrainer::ActivationType hidden_state_activation_type =
     std::get<props::HiddenStateActivation>(rnncell_props).get();
   const float dropout_rate = std::get<props::DropOutRate>(rnncell_props).get();
-  const bool integrate_bias =
-    std::get<props::IntegrateBias>(rnncell_props).get();
-  const unsigned int max_timestep =
-    std::get<props::MaxTimestep>(rnncell_props).get();
 
-  if (context.getNumInputs() != 1) {
-    throw std::invalid_argument("RNNCell layer takes only one input");
+  if (context.getNumInputs() != 2) {
+    throw std::invalid_argument("RNNCell layer expects 2 inputs but got " +
```

Comment:
It might be better to add more details like ```one for the input and hidden for the other```.

Reply:
Done